i18n/en.json

```diff
@@ -8,5 +8,9 @@
 	"wikibasereconcileedit-desc": "This is the ON OR Prototype extension",
 	"wikibasereconcileedit-editendpoint-invalid-reconcile-json": "Invalid reconcile JSON supplied",
 	"wikibasereconcileedit-editendpoint-unsupported-reconcile-version": "The only supported reconciliation {{PLURAL:$2|version is|versions are}} $1",
-	"wikibasereconcileedit-editendpoint-invalid-reconcile-propertyid": "$1 requires a single $2 key mapped to a property ID, such as P123"
+	"wikibasereconcileedit-editendpoint-invalid-reconcile-propertyid": "$1 requires a single $2 key mapped to a property ID, such as P123",
+	"wikibasereconcileedit-editendpoint-invalid-property-type-must-be-url": "urlReconcile property must be of type url",
```

Comment:
The urlReconcile name should probably be a message parameter, like $2 in the previous message.

Reply:
Done.

Reply:
Done.